### ISSUE TITLE
Stable identity of scoped action creators

### ISSUE THREAD

**Issue by inyono** [edtr-io/edtr-io#151]
Currently, scoped action creators are recreated each time which doesn't lead to correct behaviour for connected components using e.g. `React.useEffect`. Instead: scoped action creators should be created only once per scope & reused

**Comment by inyono** [comment]
No longer relevant
